src/ln/peer_handler.rs

```diff
@@ -889,11 +927,11 @@ impl<Descriptor: SocketDescriptor> PeerManager<Descriptor> {
 							let encoded_msg = encode_msg!(msg, 258);
 
 							for (ref descriptor, ref mut peer) in peers.peers.iter_mut() {
-								if !peer.channel_encryptor.is_ready_for_encryption() || peer.their_global_features.is_none() {
+								if !peer.channel_encryptor.is_ready_for_encryption() || peer.their_global_features.is_none() || peer.sync_status != msgs::InitSyncTracker::Sync(false)  {
```

Comment:
We should check more than just if we're not syncing - we should send the update if we've already gotten that far in the sync, but shouldn't send the update if its further back.

Reply:
yip